src/cmd-coreos-prune

```diff
@@ -148,16 +148,16 @@ def main():
 
                     # Skip if the action has been handled previously for the build
                     if action in previous_cleanup:
-                        # If we are in here then there has been some previous cleanup of
-                        # this type run for this build. For all types except `images` we
-                        # can just continue.
-                        if action != "images":
-                            print(f"\t\tBuild {build_id} has already had {action} pruning completed")
-                            continue
+                        print(f"action {action} is in previous cleanup section")
```

Comment:
Leftover? Seems redundant with the next print statement.

Reply:
I think it was part of my debugging - will drop it.